doc/source/serve/index.rst

```diff
@@ -36,12 +36,12 @@ Ray Serve can be used in two primary ways to deploy your models at scale:
 2. Alternatively, call them from :ref:`within your existing Python web server <serve-web-server-integration-tutorial>` using the Python-native :ref:`servehandle-api`.
 
 .. note::
-  Serve recently added an experimental first-class API for model composition (pipelines).
-  Please take a look at the :ref:`Pipeline API <serve-pipeline-api>` and try it out!
+  Serve recently added an experimental first-class API for deployment graph.
```

Comment:
```suggestion
  Serve recently added an experimental API for building deployment graphs of multiple models.
```

Maybe link "deployment graph" docs here instead of in the next sentence?

Reply:
sg